### ISSUE TITLE
PHP 8.1 Fatal error:  During inheritance of ArrayAccess in Collection.php

### ISSUE THREAD

**Issue by HepplerDotNet** [laravel/framework#39795]
- Laravel Version: 8.42.1
- PHP Version: 8.1.0
- Database Driver & Version: MariaDB 10.6

### Description:
After upgrading from PHP 8.0.13 to 8.1 I get this error:
```
PHP message: PHP Fatal error:  During inheritance of ArrayAccess: Uncaught ErrorException: Return type of Illuminate\Support\Collection::offsetExists($key) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /usr/share/nginx/html/vendor/laravel/framework/src/Illuminate/Collections/Collection.php:1443
```

**Comment by devcircus** [comment]
Looks like 8.1 support was recently completed. You'll likely need to upgrade Laravel. This specific error was fixed in 8.53 I believe, but I'd update to the latest to make sure everything has been updated for 8.1.

**Comment by nunomaduro** [comment]
Please ensure you are using the latest version of Laravel.
